### ISSUE TITLE
Printer stops all function after first few layers. Requires restart

### ISSUE THREAD

**Issue by jar0fair** [emilio1625/Anycubic-Kossel-Marlin#4]
##Bug Report
- Description: Printer is freezing up and stops all fans and motion after first layer or two of printing. LCD screen is unresponsive, requires restart.
- Expected behaviour: Complete the print
- Actual behaviour: print aborted, machine unresponsive.
- Steps to reproduce: “Print from SD”

**Comment by emilio1625** [comment]
have you tried with another SD card? does this happens also when printing by serial interface?

**Comment by jar0fair** [comment]
@emilio1625 Yes, I have printed with multiple SD cards and have attempted to print files that were printed successfully in the past, from both SDs and the issue persists. I am wondering if the PSU has failed.

**Comment by emilio1625** [comment]
Sorry, school keeps me busy, do you have any update on your problem? have you already tried to do a serial print? I would like to know if the problem is related to the LCD or to the firmware itself

**Comment by jar0fair** [comment]
@emilio1625 Wow, interesting. I just tried it, and it did print okay via USB. Oh great...now I have to troubleshoot the LCD screen

**Comment by emilio1625** [comment]
any update?
